scripts/terminalserver/terminalserver.jl

```diff
@@ -18,6 +18,24 @@ end
 include("repl.jl")
 include("../debugger/debugger.jl")
 
+function getVariables()
+    M = Main
+    variables = []
+    for n in names(M)
+        !isdefined(M, n) && continue
+        x = getfield(M, n)
+        x isa Module && continue
+        x==Main.vscodedisplay && continue
+        n==Symbol("@run") && continue
+        n==Symbol("@enter") && continue
```

Comment:
why we don't wrap those three into `_vscodeserver` ? `export @run, @enter; using _vscodeserver` won't work ?

Reply:
Yep, good idea! That won't change the need to filter them out here, right?

Should we do this in a separate PR?

Reply:
If we wrap them in `_vscodeserver`, then these condition check won't be needed since you get those names only from `Main`

Reply:
xref: #1228

Reply:
That raises a good point, actually -- we should probably have a way to switch modules (or follow the editor module).

Reply:
yeah, while I think module switching can be done in another follow up PR.

Reply:
We should probably have a tree drill down for modules in the workspace, right? Definitely a new PR, though :)

I'll do the rearrangements that @aviatesk suggested in a new PR once this one here is merged.